swagger.yaml

```
@@ -143,6 +143,42 @@ paths:
         500: 
           $ref: "#/responses/Error"
 
+  /migration-jobs/{job_id}/events:
+    get:
+      security: 
+        - Authorization: [migration:read]
+      tags:
+        - private
+      summary: "Gets a migration job's events"
+      description: >
+        "Gets a specific migration job using the unique job id identifier and returns 
```

I think it would be better not to have the words 'id' and 'identifier' alongside each other. I would change it to something like this:
```suggestion
        "Gets a specific migration job using the unique identifier (job id) and returns 
```